python/lsst/ts/standardscripts/maintel/offset_camera_hexapod.py

```diff
@@ -78,6 +78,18 @@ def get_schema(cls):
               v:
                 type: number
                 description: Ry offset (deg).
+              reset_axis:
```

Comment:
I think the naming here is backwards, `reset_only` should be the boolean and `reset_axis` the name of the axis you want to reset.

Reply:
I made it in this way to give flexibility, so the user can reset the axis first before applying the offsets (maybe this property should be called `reset_first`). For the `reset_only` property, I focused on giving flexibility to reset any axis the user may want, instead of resetting all of them (which would be case if this is a boolean).

Reply:
I refactored the code to have only the `reset_axis` property to deal with all possible scenarios to reset the axes. Thanks for your suggestions!